packages/sdk-ui-ts/src/types/token.ts

```diff
@@ -32,6 +32,9 @@ export interface CoinPriceFromInjectiveService {
 export interface TokenWithBalance extends Token {
   balance: string
   allowance: string
+  baseDenom?: string
```

Comment:
Let's use the implementation for `BankBalanceWithToken` instead of re declaring: 
```
{
    baseDenom?: string
    channelId?: string
    isCanonical?: boolean
}
```

Reply:
@ThomasRalee `BankBalanceWithToken` is a different type than what we're accomplishing with `TokenWithBalance`. `TokenWithBalance` contains an `allowance` property, which is not present on `BankBalanceWithToken`. I agree that our `token` typings could use some work, and I would like to take a crack at it in the next PR